packages/formation-react/.storybook/preview.js

```diff
@@ -57,6 +57,7 @@ export const parameters = {
     viewports,
     defaultViewport: 'small',
   },
+  viewMode: 'docs',
```

Comment:
Where on earth did you find this documentation‽

Reply:
That's a great question! I found it in this GitHub comment after searching around: https://github.com/storybookjs/storybook/issues/12111#issuecomment-678008454

However, I suspect that it should live here: https://storybook.js.org/docs/react/writing-stories/parameters#global-parameters